apps/main-landing/src/components/products-section/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,48 @@
+import { classes } from '@idriss-xyz/ui/utils';
+import Link from 'next/link';
+
+import { TabOption } from '../../types';
+
+type Properties = {
+  options: TabOption[];
+  activeOptionKey: string;
+  asLink: boolean;
+};
+
+export const Tabs = ({ activeOptionKey, options, asLink }: Properties) => {
+  return (
+    <div className="flex items-start gap-1 rounded-[50px] bg-[#022218] p-1 text-label5 lg:text-label4">
+      {options.map((option) => {
+        return asLink ? (
+          <Link
+            href={`/#${option.key}`}
+            passHref
+            legacyBehavior
+            key={option.key}
```

Comment:
you don't need children `a` and `passHref` since it's our internal navigation

Reply:
I think keeping it in that form will be better for the keyboard navigation, wdyt?

Reply:
nevermind, Link also has native tabindex